planning/behavior_path_planner/autoware_behavior_path_planner_common/src/turn_signal_decider.cpp

```diff
@@ -138,11 +138,11 @@ std::optional<TurnSignalInfo> TurnSignalDecider::getIntersectionTurnSignalInfo(
   const size_t current_seg_idx, const RouteHandler & route_handler,
   const double nearest_dist_threshold, const double nearest_yaw_threshold)
 {
-  const auto requires_turn_signal = [&](const auto & lane_attribute) {
+  const auto requires_turn_signal = [&](const auto & turn_direction, const bool is_in_turn_lane) {
```

Comment:
```suggestion
  const auto requires_turn_signal = [&current_vel](const auto & turn_direction, const bool is_in_turn_lane) {
```

Reply:
applied suggested change in https://github.com/autowarefoundation/autoware.universe/pull/9005/commits/aa288021cd0a9570d7dcd19b70c39c94b06ac4ae
and confirmed it works as expected in psim, thanks!